Makefile

```diff
@@ -2,7 +2,7 @@
 
 init:
 	pip install pipenv --upgrade
-	pipenv install --dev --skip-lock
+	pipenv install --dev --skip-lock --sequential
```

Comment:
not part of this, but should we start thinking about getting rid of pipenv @nicktimko?

Reply:
Eoghan and I were just discussing our dislike of pipenv :smile: 

I believe our Python squads have since moved on from pipenv to something else, right? I haven't kept up since moving to work on Elixir services.

Reply:
yeah, we're mostly using https://github.com/jazzband/pip-tools
I can sort it as hacktoberfest improvement.